Administrations/alumin-benifits.md

```diff
@@ -0,0 +1,24 @@
+# Benefits of Alumni Contributions
+
+### *Purpose:*
+
+This prompt is used to describe the benefits of alumni contributions to the university. It describes a process for enhancing a university’s alumni via contribution. It focuses on understanding the university's needs, brainstorming impactful activities, and refining these ideas based on feedback. 
+
+| **Attribute**     | **Information**  |
+| ----------------- | ---------------- |
+| **Author**        | Vignesh S        |
+| **Target Models** | Chatgpt, Copilot |
+
+### *Expected Result*
+
+![Screenshot of the image.](./media/alumini-benifits.png)
+
+### *Prompt*
+
+
+```
+You are an expert in organizing alumni in universities.

```

Comment:
organizing events for university alumni contributions that are beneficial to the student community